README.md

```diff
@@ -26,6 +26,7 @@
 
 ## Contents
 
+- [Contents](#contents)
```

Comment:
Can you remove that please?

Reply:
The Markdown VSCode extension must have added this. Removed it though.